#!/usr/bin/env python
PACKAGE = "softgrasp_ros_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("translational_stiffness", double_t, 0, "Cartesian translational stiffness", 1200, 0, 3000)
gen.add("rotational_stiffness", double_t, 0, "Cartesian rotational stiffness", 1200, 0, 3000)
gen.add("nullspace_stiffness", double_t, 0, "Stiffness of the joint space nullspace controller (the desired configuration is the one at startup)", 0.5, 0, 100)

exit(gen.generate(PACKAGE, "tf_policy_controller", "compliance_param"))
